.gitignore

```diff
@@ -3,3 +3,6 @@ Project/website/htmlcov
 **/.DS_Store
 Project/website/db.sqlite3
 Project/website/theory/__pycache__
+Project/website/venv
+!**/__pycache__/
+!**/migrations/
```

Comment:
This is merely a confirmation - these two statements will reinclude any of the migrations and _pycache folders in the project that were excluded by any of the above statements? Or if not, could you tell me what exactly it does?

Reply:
These two statements are my intention to exclude all __pycache__ and migration files during git commit. The statement above doesn't seem to work to exclude pycache files, so I added a new one, which seems to work.

Reply:
Okay. 2 thoughts - 
- We usually (always) want to track the migration files in a Django project. These files are notorious in the sense of making life harder in the later stages of project development. Tracking them helps maintain the order of the migrations - because they're applied one after the other. 
- The pycache folder is a good candidate for you to exclude, however since your project structure is a bit different, git might be having some trouble locating it. Additionally, "!" is a not operator which tells git to include the pattern following the "!" operator. Check this documentation for more info - [link](https://git-scm.com/docs/gitignore). Read the last example for a better clarity over the "!" operator.

Reply:
__pycache__/ seems to work. Thanks for the link and suggestions!